static/app/components/sidebar/index.tsx

```diff
@@ -417,7 +438,11 @@ function Sidebar() {
       <SidebarItem
         {...sidebarItemProps}
         icon={<IconLightning />}
-        label={<GuideAnchor target="performance">{t('Performance')}</GuideAnchor>}
+        label={
+          <GuideAnchor target="performance">
+            {hasNewNav ? truncateString(t('Performance'), 4) : t('Performance')}
```

Comment:
you can't really truncate translated text like this, either removing translations or using another truncation method like css ellipsis to truncate might be better. Should also consider the aria label once the string is truncated

Reply:
hmmm good point 🤔 
let me ask Vu how we want to address this